_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -1,25 +1,62 @@
 ---
 layout: default
-title: add_entries
+title: Add entries processor
 parent: Processors
 grand_parent: Pipelines
-nav_order: 45
+nav_order: 40
 ---
 
 # add_entries
 
-## Overview
+The `add_entries` processor adds entries to an event.
 
-The `add_entries` processor adds an entry to the event and is a [mutate event](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/mutate-event-processors#mutate-event-processors) processor. The following table describes the options you can use to configure the `add_entries` processor.
+### Configuration
 
-Option | Required | Type | Description
-:--- | :--- | :--- | :---
-entries | Yes | List | List of events to be added. Valid entries are `key`, `value`, and `overwrite_if_key_exists`.
-key | N/A | N/A | Key of the new event to be added.
-value | N/A | N/A | Value of the new entry to be added. Valid data types are strings, booleans, numbers, null, nested objects, and arrays containing the aforementioned data types.
-overwrite_if_key_exists | No | Boolean | If true, the existing value is overwritten if the key already exists within the event. Default value is `false`.
+You can configure the `add_entries` processor with the following options.
 
-<!--- ## Configuration
+| Option | Required | Description |
+| :--- | :--- | :--- |
+| `entries` | Yes | A list of entries to add to an event. |
+| `key` | Yes | The key of the new entry to be added. Some examples of keys include `my_key`, `myKey`, and `object/sub_Key`. |
+| `value` | Yes | The value of the new entry to be added. You can use the following data types: strings, Booleans, numbers, null, nested objects, and arrays. |
+| `overwrite_if_key_exists` | No | When set to `true`, the existing value is overwritten if `key` already exists in the event. The default value is `false`. |
 
-Content will be added to this section.--->
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - add_entries:
+        entries:
+        - key: "newMessage"
+          value: 3
+          overwrite_if_key_exists: true
+  sink:
+    - stdout:
+```
+{% include copy.html %}
+
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` file with that filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
+
+For example, before you run the `add_entries` processor, if the `logs_json.log` file contains the following event record:
```

Comment:
The order in wording created by splitting the example is a little confusing to me. Just a suggestion, but maybe something like:

"To provide an example, consider the following event record contained in the `logs_json.log` file before running the `add_entries` processor:"

[example]

Then you can start line 55 with:
"When you run the `add_entries` processor using the configuration shown previously, it adds a new entry..."

Global comment

Reply:
Switching the order around might do the trick there.

Reply:
I'll address this particular file and other files similarly separately since its out of scope with DP 2.2 release pending.